src/transformers/models/gemma/modeling_gemma.py

```diff
@@ -49,6 +49,7 @@
 )
 from .configuration_gemma import GemmaConfig
 
+from ..llama.modeling_llama import prepare_fa2_from_position_ids
```

Comment:
we never import from another modeling code in transformers, we use "copied from" or we define the general function in `falsh_attention_utils` for example